### PR TITLE
Now use https protocol for links to refinerycms.com

### PR DESCRIPTION
**Pull request by bricesanchez**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -33,4 +33,4 @@ inclusion.
 
 ## Links ##
 
-See also the [Contributing to Refinery](http://refinerycms.com/guides/contributing-to-refinery) guide.
+See also the [Contributing to Refinery](/guides/contributing-to-refinery) guide.
```
Comment (parndt) [inline contributing.md]: this will 404 if you're viewing this on GitHub, which you probably are 😄

Comment (bricesanchez) [reply contributing.md]: Fixed

Comment (parndt) [reply contributing.md]: I don't see the change? The link should be: `https://www.refinerycms.com/guides/contributing-to-refinery`

Comment (bricesanchez) [reply contributing.md]: Sorry it's the only one i've missed.

**Pull request merged**